### ISSUE TITLE
Running an individual S-Unit Test fails with security error -Jadeite 3.0.50

### ISSUE THREAD

**Issue by regkrock** [ericwinger/Jade#335]
Running all tests works. But running an individual test fails.

The screen shots show selecting the test, right clicking and then running the individual test.

![image](https://user-images.githubusercontent.com/7550442/49244946-01e76a80-f3df-11e8-850d-9dafd2fa17ff.png)

The test fails and with a security walkback.

![image](https://user-images.githubusercontent.com/7550442/49244964-0dd32c80-f3df-11e8-977d-8e76ece9f521.png)

However running all of the tests for the class works.

![image](https://user-images.githubusercontent.com/7550442/49245008-2d6a5500-f3df-11e8-999e-4fee27177864.png)

![image](https://user-images.githubusercontent.com/7550442/49245015-32c79f80-f3df-11e8-8241-c92d435bb32c.png)

**Comment by regkrock** [comment]
I was running as DataCurator. When I switched to SystemUser the security error was corrected.

**However, there is still a problem**

When I run the following the individual test from a workspace if fails. Also, when I run the TestCase from the workspace, the one test fails.

![image](https://user-images.githubusercontent.com/7550442/49339685-3a678e00-f603-11e8-881e-f5cf0e533e0a.png)

However when I run it from the Project Browser all the tests pass.

![image](https://user-images.githubusercontent.com/7550442/49339873-d72b2b00-f605-11e8-8389-0e7815b17e43.png)

But if I run it from the Sunit tab, the tests are correctly displayed:

![image](https://user-images.githubusercontent.com/7550442/49339895-43a62a00-f606-11e8-8eb2-65d42b47c8b7.png)
